### PR TITLE
Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by khaosdoctor**
As of #233 

Will add other files here as I translate them.

### CODE REVIEW

**Comment by khaosdoctor** [comment]
CC @orta

File: packages/tsconfig-reference/copy/en/sections/top_level.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-### Root Fields
+### Campos Raiz
 
-Starting up are the root options in the TSConfig - these options relate to how your TypeScript or JavaScript project is set up.
+Para começar, temos algumas opções na raiz do TSConfig - essas opções são relacionadas a como seu projeto TypeScript ou JavaScript está configurado.
```
Comment (orta) [inline packages/tsconfig-reference/copy/en/sections/top_level.md:3]: Careful, these are the English files ^

Comment (orta) [reply packages/tsconfig-reference/copy/en/sections/top_level.md]: I pushed a commit fixing this

**Comment by orta** [comment]
Tested locally and it works great:

Screen Shot 2020-07-16 at 11 12 46 AM

Screen Shot 2020-07-16 at 11 13 02 AM

File: packages/tsconfig-reference/copy/en/sections/top_level.md (hunk 1 above)
Comment (khaosdoctor) [reply packages/tsconfig-reference/copy/en/sections/top_level.md]: Oops! Sorry about it I didn't see it!

**Comment by khaosdoctor** [comment]
@orta I fixed a typo in 6f49278 in the English file, but had to fix conflicts and merge in 086a174. So, if you wan to cherry-pick the commit to `v2` to fix it in the online docs

File: packages/tsconfig-reference/copy/en/options/allowUnreachableCode.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ When:
 - `true` unreachable code is ignored
 - `false` raises compiler errors about unreachable code
 
-These warnings are only about code which is provably unreachable due to the use of JavaScript syntax, for example:
+These warnings are only about code which is probably unreachable due to the use of JavaScript syntax, for example:
```
Comment (orta) [inline packages/tsconfig-reference/copy/en/options/allowUnreachableCode.md:12]: This isn't a typo, it means you can prove it is unreachable - the context switch to probably makes it seem unreliable instead :D

Comment (khaosdoctor) [reply packages/tsconfig-reference/copy/en/options/allowUnreachableCode.md:12]: Oh! Sorry, I thought it was a typo. Me and my small vocab 😢 

I'll reverse it!

**Comment by khaosdoctor** [comment]
@orta If that's ok, you can merge this PR. I was thinking it's going to be very complex to conduct the translation if other people join me with only one PR, so I'll be opening other PRs in a "bunch-of-files"-base. Translating chunk by chunk instead of it all

**Comment by orta** [comment]
I'd like another active speaker to give a review that it feels alright (maybe @alvarocamillont ?) - then yep, totally good to go

**Comment by alvarocamillont** [comment]
That will be great!

**Comment by khaosdoctor** [comment]
@alvarocamillont so, can you please review this PR? Then with your OK we merge it and make it easier for others to contribute :)

**Comment by alvarocamillont** [comment]
OK :)

**Comment by alvarocamillont** [comment]
Hello @orta  ! I'm trying to build on my local machine but it keeps giving yarn error ...
My machine is linux (Ubuntu) with node 14 and yarn 2.1.1, any tips?

**Comment by orta** [comment]
is it https://github.com/microsoft/TypeScript-Website/issues/792#issuecomment-662038631 ?

**Comment by alvarocamillont** [comment]
It is not exactly that error, but it is very similar. It is in the same step in yarn install.
I am validating the text in the md file since you successfully built it. But I would like to run it locally, besides after this validation of the khaosdoctor file I want to create my PRs;)

File: packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+oneline: "Exibe um erro quando o código nunca será executado"
+---
+
+Quando:
+
+- `undefined` (padrão) dá sugestões como avisos para o editor
+- `true` todo o código não executável é ignorado
+- `false` exibe um erro de compilação quando código não executável é detectado
+
+Mude para false para desabilitar os avisos sobre código não executável.
```
Comment (alvarocamillont) [inline packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md:12]: `false`

Comment (alvarocamillont) [reply packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md:12]: faltou o ` `

File: packages/tsconfig-reference/copy/pt/options/allowUnusedLabels.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+display: "Permitir Labels Não Utilizadas"
+oneline: "Exibe um erro quando uma label é criada acidentalmente"
+---
+
+Defina como false para desabilitar os avisos sobre labels não utilizadas.
```
Comment (alvarocamillont) [inline packages/tsconfig-reference/copy/pt/options/allowUnusedLabels.md:6]: Faltou o ` `

**Review by alvarocamillont — CHANGES_REQUESTED** [review]

**Comment by orta** [comment]
Can you get me a log (or tell me if it's a specific dep) I'm trying to clean the tree a bit to see if I can drop them

**Comment by alvarocamillont** [comment]
Sure! Do you want me to pass this issue or the issue you indicated?

**Review by danilofuchs — APPROVED** [review]

File: packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md (hunk 3 above)
Comment (khaosdoctor) [reply packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md:12]: Eu segui a documentação de acordo com  oque estava lá, no meio do texto ele não estava usando o `false` só quando se referia à palavra mesmo

Comment (alvarocamillont) [reply packages/tsconfig-reference/copy/pt/options/allowUnreachableCode.md:12]: Então tudo bem :)

**Review by alvarocamillont — APPROVED** [review]

**Comment by orta** [comment]
thanks folks, will merge on green!

**Comment by orta** [comment]
Alright, assuming https://github.com/microsoft/TypeScript-Website/actions/runs/179090970 goes well then it should show up in https://www.staging-typescript.org/pt/tsconfig

**Pull request merged**